coverage.gradle

```diff
@@ -18,11 +21,13 @@ def initializeReport(report, projects, classExcludes) {
             switch (project) {
                 case { project.plugins.hasPlugin("com.android.application") }:
                     androidClassDirs.add("${project.buildDir}/tmp/kotlin-classes/gplayDebug")
+                    androidSourceDirs.add("${project.buildDir}/generated/source/kapt/gplayDebug")
```

Comment:
Without this we have some generated files with coverage percentages but no source to look at. Maybe we should ignore them from the coverage instead?

Reply:
I would be for ignoring all generated code (since we're technically not the author~)